Makefile

```diff
@@ -37,6 +40,20 @@ stop-airflow2:
 	@ echo "$(BUILD_PRINT)Stopping the AirFlow services"
 	@ docker-compose --file ./airflow2/docker-compose.yml --env-file ../.env down
 
+start-tika: build-volumes start-storage
+	@ echo "$(BUILD_PRINT)Starting Apache Tika"
+	@ echo "$(BUILD_PRINT)Warning: the shared folders need R/W permissions"
+	@ docker-compose --file docker-compose.yml --env-file .env up -d apache_tika
+
+stop-tika:
+	@ echo "$(BUILD_PRINT)Stopping Apache Tika"
+	@ docker-compose --file docker-compose.yml --env-file .env down
+
+start-elk: build-volumes start-storage
```

Comment:
let's avoid specifying the services, in the command line. The reason for multiple compose files is to group the services that should start together. If we need a separation between tika and elk, we shall do so.

Reply:
I've removed the Makefile code.